src/services/local-storage.js

```diff
@@ -0,0 +1,24 @@
+const KEY = "credentials";
```

Comment:
in Java this would be a `private` constant to the `LocalStorage` class. In pure js we just get rid of the boilerplate :)